fission/src/Synthesis.tsx

```diff
@@ -49,12 +49,13 @@ import ScoringZonesPanel from "@/panels/configuring/scoring/ScoringZonesPanel"
 import ZoneConfigPanel from "@/panels/configuring/scoring/ZoneConfigPanel"
 import ScoreboardPanel from "@/panels/information/ScoreboardPanel"
 import DriverStationPanel from "@/panels/simulation/DriverStationPanel"
-import ManageAssembliesModal from '@/modals/spawning/ManageAssembliesModal.tsx';
-import World from '@/systems/World.ts';
-import { AddRobotsModal, AddFieldsModal, SpawningModal } from '@/modals/spawning/SpawningModals.tsx';
-import ImportMirabufModal from '@/modals/mirabuf/ImportMirabufModal.tsx';
-import Skybox from './ui/components/Skybox.tsx';
-import ImportLocalMirabufModal from '@/modals/mirabuf/ImportLocalMirabufModal.tsx';
+import ManageAssembliesModal from "@/modals/spawning/ManageAssembliesModal.tsx"
+import World from "@/systems/World.ts"
+import { AddRobotsModal, AddFieldsModal, SpawningModal } from "@/modals/spawning/SpawningModals.tsx"
+import ImportMirabufModal from "@/modals/mirabuf/ImportMirabufModal.tsx"
+import Skybox from "./ui/components/Skybox.tsx"
+import ImportLocalMirabufModal from "@/modals/mirabuf/ImportLocalMirabufModal.tsx"
+import PokerPanel from "./panels/PokerPanel.tsx"
```

Comment:
This is currently in src/panels
All the other panels are in @/panels ----- src/ui/panels

Reply:
Fun, this predates that move. Thanks for catching that